main.cpp

```diff
@@ -4,7 +4,20 @@ using namespace std;
 
 int main()
```

Comment:
I just saw your facebook message. I get it now. Will make the necessary changes.